deploy/kubedirector/kubedirector_v1alpha1_kubedirectorcluster_crd.yaml

```diff
@@ -10,10 +10,19 @@ spec:
     plural: kubedirectorclusters
     singular: kubedirectorcluster
   scope: Namespaced
-  version: v1alpha1
+  subresources:
+    status: {}
```

Comment:
I'm not certain this is correct. We haven't been using status as an explicit "subresource", but rather just as a property of the object. Initially this was because we were running on at least some K8s versions that didn't support custom CR subresources. Cf. https://github.com/bluek8s/kubedirector/issues/37

We might be able to move to using a subresource now, but I'm not sure what code changes might then be required.

Reply:
Those lines were added by the operator-sdk. We curently don't have any KubeDirectorCluster subresources so I will remove them.

Reply:
After further testing I've found that I had to add these lines back. I was unable to create KubeDirectorCluster's when they were deleted.